### ISSUE TITLE
Any plan to support 2 or 3 buttons nav bar?

### ISSUE THREAD

**Issue by on6hz** [Magisk-Modules-Alt-Repo/CustomizedNavbar#4]

**Comment by PycmShoma** [comment]
Hi.
What exactly are you interested in? Color of the navbar buttons in the system accent color?
I have been using gestures since Android OS 10. I don’t know how it works with the buttons now, but if I have free time I can look.
Again, there may be a problem in the compatibility of the mod on all firmware and Android OS versions. Need to watch.

**Comment by on6hz** [comment]
Hi, normally the navbar theme are followed current app or keyboard, but there are quite some apps and soft keyboard which are highly dependent for me, are showed the default, black themed navbar, in short, the black background navbar is a nightmare under the Inverted Color from the system accessibility, and hard to tackle with.

A simply white background for those apps and keyboard could end this nightmare but unfortunately I could not archive it, over recent years cope with. 

Basically I use AOSP ROMs, and follow the majority Stable OS updates.

**Comment by gzyguo** [comment]
![Uploading PXL_20221127_113814727.jpg…]()

**Comment by on6hz** [comment]
![trashed-1672112277-PXL_20221127_113751926](https://user-images.githubusercontent.com/14918112/204118219-aa93aa0b-9f6c-442d-b0b7-f748d5e235ff.jpg)
![PXL_20221127_113900832](https://user-images.githubusercontent.com/14918112/204118221-0542c4e5-c14c-4762-8ce6-f78da78134c6.jpg)

**Comment by PycmShoma** [comment]
so you have styles in the firmware so written that the statusbar and navbar have a theme inversion.
Usually in AOSP the color of the statusbar and navbar parent the theme
![Screenshot_20221128-001947_Настройки](https://user-images.githubusercontent.com/6963578/204160370-ae672bed-1769-49de-8238-d7d2ee68a9f6.png)

![Screenshot_20221128-001936_Chrome](https://user-images.githubusercontent.com/6963578/204160379-1f3d92de-3a61-47b4-9b8d-0dde6ece5910.png)

**Comment by on6hz** [comment]
Sorry I don't really understand your text, the firmware is the OS right? The inversion is a system accesibility options, not really a theme, which in somewhat hardware level I think which is hold by most Android variances even in Windows.

Yes, the reason is that some apps and keyboard doesn't change the navbar, which simply leave it black.

And if the status bar is off the topic, I can tolerate it, since it's far from my eyes compare to the navbar and it has less area than it, less hurt my eyes.

**Comment by PycmShoma** [comment]
I'm afraid that this cannot be fixed with a universal patch, and I do not deal with local ones for one specific OS.  Sorry
